doc/classes/RenderingServer.xml

```diff
@@ -3163,6 +3163,14 @@
 				If [code]true[/code], reflections will ignore sky contribution. Equivalent to [member ReflectionProbe.interior].
 			</description>
 		</method>
+		<method name="reflection_probe_set_blend_distance">
```

Comment:
Not sure why the workflow run is giving this error here:
`doc/classes/RenderingServer.xml:3166: element method: Schemas validity error : Element 'method': Character content other than whitespace is not allowed because the content type is 'element-only'.`